lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -855,6 +863,8 @@ getLoopControl(const BranchInst *Branch, std::vector<SPIRVWord> &Parameters,
       Parameters.push_back(ArraySflnPair.first);
       Parameters.push_back(ArraySflnPair.second);
     }
+    BM->addExtension(ExtensionID::SPV_INTEL_fpga_loop_controls);
```

Comment:
@MrSidims, you are adding `SPV_INTEL_fpga_loop_controls`  capability here if `DependencyArrayParameters` is not empty. But this might be the case if `"llvm.loop.unroll.disable"` metadata was encountered (for example), which is not part of `SPV_INTEL_fpga_loop_controls`

Reply:
All ok